pkg/scheduler/scheduler_test.go

```diff
@@ -896,6 +921,11 @@ func TestSchedule(t *testing.T) {
 			},
 		},
 		"no overadmission while borrowing": {
+			// we disable this test for MultiplePreemption
+			// logic, as the new logic considers this
+			// unschedulable, while the old logic considers
+			// it skipped.
```

Comment:
so what changes is that eng-gamma would not be in `wantLeft`? That's fine, but I wonder if we care about duplicating the test case for it.
Maybe not worth, but instead we can update this test when we remove the Legacy mode.

Reply:
exactly, but rather in `wantInadmissibleLeft`. I added a duplicate test case, with that minor change, to cover the new logic.

as a follow up PR, we could update the legacy logic to treat this case as `inadmissible` as well.